report_alternative_layout/report/report_template.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="external_layout_standard" inherit_id="web.external_layout_standard">
+        <!-- Hide company address in the header if alternative layout is selected -->
+        <xpath expr="//div[@name='company_address']/.." position="attributes">
+            <attribute name="t-if">not env.context.get('apply_alternative_layout')</attribute>
+        </xpath>
+        <!-- Show company address in the body if alternative layout is selected -->
+        <xpath expr="//t[@t-call='web.address_layout']" position="attributes">
+            <attribute name="t-if">not env.context.get('apply_alternative_layout')</attribute>
+        </xpath>
+        <xpath expr="//t[@t-call='web.address_layout']" position="after">
+            <div t-else="" class="row" name="address_layout">
+                <div class="col-7" name="partner_address">
+                    <div t-set="partner_name" t-value="o.env['ir.actions.report']._get_partner(xmlid, o.partner_id)"/>
```

Comment:
I haven't checked the functionality. But is this the correct usage of` t-set`? Should we use `<t t-set>` instead of `<div t-set>`, right?

Reply:
I confirmed functionality and odoo standard code.  We should use <t t-set>, so I updated.